scripts/container_ops.sh

```diff
@@ -64,39 +71,73 @@ run_cmd() {
 ###################################################################
 
 # >>>>>>>  Initialization
+# Input vars
+REGISTRY_LOGIN=${1:-""}
+PUSH_TO_REGISTRY=${2:-""}
```

Comment:
This two parameters are "boolean" so default value should be boolean as well => true / false

Reply:
BTW, you don't need both parameter because `push_to_registry` means that we must be logged in

Reply:
PUSH_TO_REGISTRY will remain and REGISTRY_LOGIN will be removed as suggested. 
Default value will be set to false.